translator/trace/jaeger/traces_to_jaegerproto.go

```diff
@@ -115,42 +115,64 @@ func resourceToJaegerProtoProcess(resource pdata.Resource) *model.Process {
 	}
 
 	process := model.Process{}
+	attrsCount := attrs.Cap()
 	if serviceName, ok := attrs.Get(conventions.AttributeServiceName); ok {
 		process.ServiceName = serviceName.StringVal()
-		attrs.Delete(conventions.AttributeServiceName)
+		attrsCount--
```

Comment:
Also not sure I understand why do you call `attrsCount--`. If the attrs will still have `service.name` in the `appendTagsFromResourceAttributes` you will append the `service.name` as well so not sure then why allocating only a slice of (attrs.Cap() - 1).

Reply:
`appendTagsFromResourceAttributes` does have the logic to skip the service name attribute https://github.com/open-telemetry/opentelemetry-collector/pull/906/files#diff-645cbe27bc5be6692575be50e2b28640R139

Reply:
Saw that after.